src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj

```diff
@@ -548,4 +548,11 @@
   <ItemGroup>
     <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Are changes to the imagemanifest intentional?

Reply:
Intentional. I decided to leave it there because this fixes a warning when you're building NuGet.sln in Visual Studio.

Reply:
I noticed this warning several times and I had no clue what to do about it. Now I understand it is an issue with our repo. Thanks for fixing it. I would have fixed it in another PR, but I guess it's okay.

Reply:
I will address this in another PR.

Reply:
https://github.com/NuGet/Client.Engineering/issues/1872